app/components/instant-article-field/instant-article-config.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module('cms.instant_article.config', []).factory('InstantArticleConfig', [function () {
+  var supportedFeatureTypes = [];
+
+  var factory = {};
+
+  factory.setSupportedFeatureTypes = function(featureTypes) {
+    supportedFeatureTypes = featureTypes;
+
+    return this;
+  };
+
+  factory.getSupportedFeatureTypes = function() {
+    return supportedFeatureTypes;
+  };
+
+  return factory;
+}]);
```

Comment:
Use the same `provider` config pattern as in [tunic-config.js](https://github.com/theonion/bulbs-cms/blob/6295eaf1e91db7a37cd8d5e12bc61dd016f088c8/app/shared/tunic/tunic-config.js).

Otherwise, you won't be able to access this in the `config` step of other modules since `factory` functions I don't believe are available at `config` time.